### PR TITLE
Add Remove Order process(es) modal. Order process list selectable

### PR DESCRIPTION
**Pull request by lgalis**
https://projects.engineering.redhat.com/browse/SSP-1712

Add remove order process modal
Order process list selectable.
More tests

Screenshot from 2020-08-06 18-37-03
Screenshot from 2020-08-06 18-43-00
Screenshot from 2020-08-06 18-43-13
Screenshot from 2020-08-07 11-13-41

### CODE REVIEW

File: src/smart-components/order-process/order-processes.js [hunk 1]
```diff
@@ -135,20 +141,55 @@ const OrderProcesses = () => {
...
+    }
+  ];
+
   const onSort = (_e, index, direction, { property }) => {
     dispatch(sortOrderProcesses({ index, direction, property }));
     return updateOrderProcesses();
   };
 
+  const setCheckedItems = (checkedOrderProcesses) =>
+    setSelectedOrderProcesses(checkedOrderProcesses.map((wf) => wf.id));
```
Comment (Hyperkid123) [inline src/smart-components/order-process/order-processes.js:189]: workflows reference in the mao argument

File: src/utilities/use-order-process.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { shallowEqual, useSelector } from 'react-redux';
+
+const useOrderProcess = (id) => {
+  const { orderProcesses } = useSelector(
+    ({ orderProcessReducer: { orderProcesses } }) => ({ orderProcesses }),
+    shallowEqual
+  );
+
+  return (
+    orderProcesses &&
```
Comment (Hyperkid123) [inline src/utilities/use-order-process.js:10]: you could simplify this return value with optional chaining
```jsx
orderProcessess?.data?.find(...)
```

**Comment by Hyperkid123** [comment]
Also, the bulk selection is lost from the table after changing a page

op-no-p

**Comment by lgalis** [comment]
@Hyperkid, yes - I have not fixed the selection yet. Marking draft until then.

**Comment by Hyperkid123** [comment]
The insights API does not work on ci/qa so I can't really check it in UI. The code looks good, but I want to double-check it in UI.

**Pull request merged**